scripts/spectrum-vars.js

```diff
@@ -126,6 +144,49 @@ cores.forEach(async (core) => {
     processes.push(processCSS(srcPath, dstPath, 'typography'));
 }
 
+{
+    // Typography
+    const typographyPath = path.join(
+        __dirname,
+        '..',
+        'node_modules',
+        '@spectrum-css',
+        'typography'
+    );
+
+    // Commons
+    const commonsPath = path.join(
+        __dirname,
+        '..',
+        'node_modules',
+        '@spectrum-css',
+        'commons'
+    );
+
+    // typography.css
+    {
+        const srcPath = path.join(typographyPath, 'dist', 'index-vars.css');
+        const dstPath = path.resolve(
+            path.join(__dirname, '..', 'packages', 'styles', 'typography.css')
+        );
+        console.log(`processing typography`);
+        processes.push(processCSS(srcPath, dstPath, 'typography'));
+    }
+
+    // fonts.css (2 sources so a little tricky)
+    {
+        const srcPath1 = path.join(typographyPath, 'font.css');
+        const srcPath2 = path.join(commonsPath, 'fonts.css');
```

Comment:
Now that I'm looking at the output again, I'll swap these so the variables are defined before they're used...